com.microsoft.mrtk.graphicstools.unity/Samples~/MeshOutlines/Textures/leCat_eyes_tex.jpg.meta

```diff
@@ -0,0 +1,98 @@
+fileFormatVersion: 2
```

Comment:
nit: would you mind changing these texture names to Pascal case to match the other textures?